### ISSUE TITLE
Uncaught TypeError: Cannot read property 'send' of undefined

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#2558]
1. Go to https://demo.uwazi.io/es/library/map?q=(order:desc,sort:metadata.date_published)
2. Click on a marker
3. Edit the entity
4. When saving you get the following message:
```
vendor.8e8db4cad5b99fce5757.js:2862 Uncaught TypeError: Cannot read property 'send' of undefined
    at i.send (vendor.8e8db4cad5b99fce5757.js:2862)
    at t._updateWorkerData (vendor.8e8db4cad5b99fce5757.js:2862)
    at t.load (vendor.8e8db4cad5b99fce5757.js:2862)
    at t.onAdd (vendor.8e8db4cad5b99fce5757.js:2862)
    at t.onAdd (vendor.8e8db4cad5b99fce5757.js:2862)
    at t.addSource (vendor.8e8db4cad5b99fce5757.js:2862)
    at t._load (vendor.8e8db4cad5b99fce5757.js:2862)
    at vendor.8e8db4cad5b99fce5757.js:2862
```
and probably related to this server side error:
```
[demo] ERROR Failed to index document 5ba2ba741ce4f646346b0da4: {
 "type": "mapper_parsing_exception",
 "reason": "failed to parse",
 "caused_by": {
  "type": "parse_exception",
  "reason": "field must be either [lat], [lon] or [geohash]"
 }
}
```

**Comment by daneryl** [comment]
@txau if this is related to the second error, we are going to fix it #2594, but if this has already wrong values on the db we should manually remove/fix them or do some kind of migration, i have encountered this on other instances also.

**Comment by daneryl** [comment]
@txau, do we have any report of this happening still ? validation should prevent bad values on the db, can we close ?
